env/dev/env/config.cljs

```diff
@@ -1,6 +1,5 @@
 (ns env.config)
 
-(def figwheel-urls {
-                    :ios "ws://localhost:3449/figwheel-ws"
-                    :android "ws://10.0.3.2:3449/figwheel-ws"
-                    })
\ No newline at end of file
+(def figwheel-urls {:android "ws://192.168.10.203:3449/figwheel-ws",
+ :ios "ws://localhost:3449/figwheel-ws"}
```

Comment:
Formatting seems weird, that's how Cursive formats this:
```
(def figwheel-urls {:android "ws://192.168.0.104:3449/figwheel-ws"
                    :ios     "ws://192.168.0.104:3449/figwheel-ws"})
```

Reply:
Do we need this change at all?

Reply:
Right, i usually don't commit this file.